docs/modules/ROOT/pages/hazelcast-platform-operator-wan-replication.adoc

```diff
@@ -0,0 +1,433 @@
+npm install
+npm start fill
+----
+--
+
+Go::
++
+--
+[source, bash]
+----
+cd clients/go
+go run main.go fill
+----
+--
+
+Python::
++
+--
+[source, bash]
+----
+cd clients/python
+pip install -r requirements.txt
+python main.py fill
+----
+--
+
+====
++
+You should see the following output.
++
+[source, shell]
+----
+Successful connection!
+Starting to fill the map with random entries.
+Current map size: 2
+Current map size: 3
+Current map size: 4
+Current map size: 5
+Current map size: 6
+Current map size: 7
+Current map size: 8
+Current map size: 9
+Current map size: 10
+----
+
+== Step 3. Check if map values are present in both clusters
+
+. Verify size of the map
++
+Configure the Hazelcast client to connect to the second cluster address as you did in <<Configure-Client, Configure the Hazelcast Client>>.
++
+[tabs]
+====
+
+Java::
++
+--
+[source, java]
+----
+ClientConfig config = new ClientConfig();
+config.getNetworkConfig().addAddress("<SECOND-CLUSTER-SERVICE-IP>");
+----
+--
+
+NodeJS::
++
+--
+[source, javascript]
+----
+const { Client } = require('hazelcast-client');
+
+const clientConfig = {
+    network: {
+        clusterMembers: [
+            '<SECOND-CLUSTER-SERVICE-IP>'
+        ]
+    }
+};
+const client = await Client.newHazelcastClient(clientConfig);
+----
+--
+
+Go::
++
+--
+[source, go]
+----
+import (
+	"log"
+
+	"github.com/hazelcast/hazelcast-go-client"
+)
+
+func main() {
+	config := hazelcast.Config{}
+	cc := &config.Cluster
+	cc.Network.SetAddresses("<SECOND-CLUSTER-SERVICE-IP>")
+	ctx := context.TODO()
+	client, err := hazelcast.StartNewClientWithConfig(ctx, config)
+	if err != nil {
+		panic(err)
+	}
+}
+----
+--
+
+Python::
++
+--
+[source, python]
+----
+import logging
+import hazelcast
+
+logging.basicConfig(level=logging.INFO)
+
+client = hazelcast.HazelcastClient(
+    cluster_members=["<SECOND-CLUSTER-SERVICE-IP>"],
+    use_public_ip=True,
+)
+----
+--
+====
++
+Now you can start the application to check the map size and see if the replication was successful.
++
+[tabs]
+====
+
+Java::
++
+--
+[source, bash]
+----
+cd clients/java
+mvn package
+java -jar target/*jar-with-dependencies*.jar size
+----
+--
+
+NodeJS::
++
+--
+[source, bash]
+----
+cd clients/nodejs
+npm install
+npm start size
+----
+--
+
+Go::
++
+--
+[source, bash]
+----
+cd clients/go
+go run main.go size
+----
+--
+
+Python::
++
+--
+[source, bash]
+----
+cd clients/python
+pip install -r requirements.txt
+python main.py size
+----
+--
+
+====
++
+You should see the following output.
++
+[source, shell]
+----
+Successful connection!
+Current map size: 12
+----
+
+== Clean Up
+
+To clean up the created resources remove the all Custom Resources and PVCs.
+
+[source, shell]
+----
+kubectl delete secret hazelcast-license-key
+kubectl delete $(kubectl get wanreplications,map,hazelcast -o name)
+kubectl delete pvc -l "app.kubernetes.io/managed-by=hazelcast-platform-operator"
+----
+
+== See Also
+
+- xref:operator:ROOT:backup-restore.adoc[]
```

Comment:
I checked [The WAN replication doc PR](https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/42).  It is not merged but it seems the page name will be `wan-replication`.
```suggestion
- xref:operator:ROOT:wan-replication.adoc[]
```

Reply:
Fixed.